nemo/collections/asr/parts/utils/offline_clustering.py

```diff
@@ -602,14 +666,14 @@ def getEnhancedSpeakerCount(
             max_num_speakers=emb.shape[0],
             max_rp_threshold=0.15,
             sparse_search=True,
-            sparse_search_volume=50,
+            sparse_search_volume=10,
```

Comment:
n00b question: was this just reduced for performance?

Reply:
There is a chance that `sparse_search_volume` could deteriorate the performance if it is too low.